projects/core/services/color-scheme.service.ts

```diff
@@ -0,0 +1,20 @@
+import {Inject, Injectable} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {fromEvent, Observable} from 'rxjs';
+import {map, share, startWith} from 'rxjs/operators';
+
+@Injectable({
+    providedIn: `root`,
+})
+export class TuiColorSchemeService extends Observable<boolean> {
```

Comment:
Why change of name if it's still boolean?

Reply:
otherwise there will be two services with similar names

`TuiThemeNightService` and `TuiNightThemeService`

Reply:
That's ok, they are in different packages and one is deprecated (and is just an alias for the other). We've had that before with `TUI_SANITIZER`.

Reply:
fixed